### PR TITLE
Mobile version

### PR DESCRIPTION
**Pull request by dgsierra**
-Add navbar
-Add section Welcome
-Add icons
-Add styles to the elements
-Add links to the icons
-Style background

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ To get a local copy up and running follow these simple example steps:
```
Comment (Ismailco) [inline README.md:28]: - [ ] [OPTIONAL] Your title has a typo `Mobil version`, you may consider fixing it so your PR looks professional

File: index.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:ital@1&display=swap" rel="stylesheet">
+    <title>Daniel`s Portafolio</title>
+
+</head>
+<body>
+    <nav class="nav--cointainer">
+        <a href="index.html" id="logo">Jhon Doe</a>
+        <img id="menu" src="bars-solid.svg" alt="menu icon">
+    </nav>
```
Comment (Ismailco) [inline index.html]: - [ ] This section should be wrapped in `<header>` instead, kindly implement this so your code is organized

File: index.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:ital@1&display=swap" rel="stylesheet">
+    <title>Daniel`s Portafolio</title>
+
+</head>
+<body>
+    <nav class="nav--cointainer">
+        <a href="index.html" id="logo">Jhon Doe</a>
+        <img id="menu" src="bars-solid.svg" alt="menu icon">
```
Comment (Ismailco) [inline index.html]: - [ ] The hamburger menu should have a `nav` tag, kindly consider adding the `nav` tag, the hamburger menu will contain links in the future

File: index.html [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+            <img src="./icons/Github.svg" alt="github">
+
+            <img src="./icons/LinkedIn.svg" alt="linkedin">
+
+            <img src="./icons/Angellist.svg" alt="angellist">
+
+            <img src="./icons/Twitter.svg" alt="twitter">
+
+            <img src="./icons/Medium.svg" alt="">
+        </div>
```
Comment (Ismailco) [inline index.html]: - [ ] This list of icons should be wrapped in a `ul` tag, and every icon should be inside a `li` tag, also the icons are links, so they need to be inside an `a` tag, kindly implement this so your icons work as links for external websites

File: style.css [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+  text-shadow: (0, 0, 0, 0.25);
+  line-height: 24px;
+  padding: 6px;
+}
+
+.icons {
+  align-items: baseline;
+}
+
+.welcome {
```
Comment (Ismailco) [inline style.css:78]: - [ ] The content of this section should be aligned with flexbox, kindly consider using it for styling this section

File: style.css [hunk 6]
```diff
@@ -0,0 +1,77 @@
+* {
```
Comment (Ismailco) [inline style.css:1]: - [ ] The design should look similar to the Figma one, kindly consider adding a height of 100vh to the headline section so it looks like the Figma design

|Your design|Figma design|
|-|-|
|Screenshot_20220630_044728|Screenshot_20220629_210335|

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @dgonzalesi,

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShahierNashaat — APPROVED** [review]
Hi @dgonzalesi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**